episodes/3-monitor-the-model.Rmd

```diff
@@ -359,19 +359,20 @@ We will create a function for it, because we will make use of this more often in
 import seaborn as sns
 import matplotlib.pyplot as plt
 
-def plot_history(metrics):
+def plot_history(history, metrics):
     """
     Plot the training history
 
     Args:
-        metrics(str, list): Metric or a list of metrics to plot
+        history (keras History object that is returned by model.fit())
+        metrics (str, list): Metric or a list of metrics to plot
     """
     history_df = pd.DataFrame.from_dict(history.history)
     sns.lineplot(data=history_df[metrics])
     plt.xlabel("epochs")
     plt.ylabel("RMSE")
```

Comment:
```suggestion
    plt.ylabel("metric")
```

We may not always want to use RMSE as the metric. Better to be more generic?

Reply:
Yes, in this specific notebook it is always RMSE, but you are right that it is nicer to make it more generic.